typescript/src/autolinker.ts

```diff
@@ -0,0 +1,44 @@
+import Autolinker from 'autolinker';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/betagouv/aplus/pullRequest?prid=7861423)
```suggestion
import Autolinker from "autolinker";
```